adapter.go

```diff
@@ -182,6 +197,37 @@ func NewAdapterByDBUseTableName(db *gorm.DB, prefix string, tableName string) (*
 	return a, nil
 }
 
+/*
+multiple databases support
+Example usage:
+	dbPool,err := initDbResolver([]gorm.Dialector{mysql.Open(dsn),mysql.Open(dsn2)},[]string{"casbin1","casbin2"})
+	a := initAdapterWithGormInstanceByMulDb(t,dbPool,"casbin1","","casbin_rule1")
+	a = initAdapterWithGormInstanceByMulDb(t,dbPool,"casbin2","","casbin_rule2")
+*/
+func initDbResolver(dbArr []gorm.Dialector, dbNames []string) (DbPool, error) {
+	if len(dbArr) == 0 {
+		panic("dbArr len is 0")
+	}
+	source, e := gorm.Open(dbArr[0])
+	if e != nil {
+		panic(e.Error())
+	}
+	p := specificPolicy{connnIndex: 0}
+	err := source.Use(dbresolver.Register(dbresolver.Config{Policy: &p, Sources: dbArr}))
+	var dbMap map[string]int
+	dbMap = make(map[string]int, 3)
+	for i := 0; i < len(dbNames); i++ {
+		dbMap[dbNames[i]] = i
+	}
+	return DbPool{dbMap: dbMap, policy: &p, db: source}, err
+}
```

Comment:
gofmt

Reply:
I tried using gofmt，but nothing changed here.What's the problem here?

Reply:
new line between functions

Reply:
got it

Reply:
@tangyang9464 `gofmt -w .` or `go fmt .`, you could refer to `go help fmt`